### PR TITLE
feature/FOUR-15039: The styles of the options in Welcome Screen are not the same as in figma

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
1. The order of the options
2. The spacing between options
3. The width of the tool tip is much narrower

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-15039

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy
ci:package/dynamic-ui:feature/FOUR-15039

### CODE REVIEW

File: src/components/renderer/form-list-table.vue [hunk 1]
```diff
@@ -66,9 +63,12 @@
                 <i class="fas fa-circle mr-2" />{{ $t("Completed") }}
               </b-dropdown-item>
               <b-dropdown-item
-                @click="handleDropdownSelection('requests_dropdown', 'all')"
+                variant="success"
+                @click="
+                  handleDropdownSelection('requests_dropdown', 'In Progress')
+                "
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:69]: in a single line

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-49807424d5.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-49807424d5.engk8s.processmaker.net

File: src/components/renderer/form-list-table.vue [hunk 2]
```diff
@@ -327,4 +357,29 @@ export default {
...
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.02em;
+  text-align: left;
+}
+</style>
+
+<style lang="scss">
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:372]: other?

**Review by fagubla — CHANGES_REQUESTED** [review]

File: src/components/renderer/form-list-table.vue (hunk 2 above)
Comment (luNunezProcessmaker) [reply src/components/renderer/form-list-table.vue:377]: the first one is scoped, and the b-dropdown doesn't use styles

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-49807424d5.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-49807424d5.engk8s.processmaker.net

**Pull request merged**